node/ltnodeclient.js

```diff
@@ -93,10 +93,12 @@ function sbRequire(require, resolve, path) {
   if(!getSB(path)) {
     if(path.match(/[\\\/]/) && fpath.extname(path) == ".js") {
       var sb = sandboxes[path.toLowerCase()] = context(path);
```

Comment:
@rbecheras - what would the simplest way to reproduce the error to make sure this PR fixes it? This linked issue, https://github.com/LightTable/LightTable/issues/2146, gave some steps... should that suffice?